packages/runtime/plugin-garfish/src/cli/index.ts

```diff
@@ -205,6 +206,9 @@ export const garfishPlugin = ({
                   });
                 }
               }
+              if (!preConfig.output!.uniqueName) {
```

Comment:
Using `chain.output.get('uniqueName')` is better, `chain.toConfig()` will convert the whole config object and  is slower.

Reply:
ok